test/scrollable.html

```diff
@@ -0,0 +1,148 @@
+        </paper-tabs>
+      </template>
+    </test-fixture>
+
+    <test-fixture id="zoomed">
+      <template>
+        <paper-tabs scrollable style="width: 500px;zoom: 110%;">
+          <paper-tab style="width: 150px">ITEM ONE</paper-tab>
+          <paper-tab style="width: 150px">ITEM TWO</paper-tab>
+          <paper-tab style="width: 150px">ITEM THREE</paper-tab>
+          <paper-tab style="width: 150px">ITEM FOUR</paper-tab>
+        </paper-tabs>
+      </template>
+    </test-fixture>
+
+    <script>
+
+      suite('scrollables suite', function() {
+      	
+    	var tabs, tabsContainer, leftScrollbutton, rightScrollbutton;
+      
+        suite('non-scrollable paper-tabs', function() {
+  
+          setup(function () {
+            tabs = fixture('basic');
+            leftScrollbutton = Polymer.dom(tabs.root).querySelectorAll('paper-icon-button')[0];
+            rightScrollbutton = Polymer.dom(tabs.root).querySelectorAll('paper-icon-button')[1];
+            tabsContainer = tabs.$.tabsContainer;
+          });
+          
+          test('have no scrollbuttons', function() {
+            assert.isTrue(leftScrollbutton.offsetWidth===0);
+            assert.isTrue(rightScrollbutton.offsetWidth===0);
+          });
+  
+        });
+      
+        suite('scrollable paper-tabs', function() {
+  
+          setup(function () {
+            tabs = fixture('scrollable');
+            leftScrollbutton = Polymer.dom(tabs.root).querySelectorAll('paper-icon-button')[0];
+            rightScrollbutton = Polymer.dom(tabs.root).querySelectorAll('paper-icon-button')[1];
+            tabsContainer = tabs.$.tabsContainer;
+          });
+  
+          test('have scrollbuttons', function() {
+            assert.isTrue(leftScrollbutton.offsetWidth>0);
+            assert.isTrue(rightScrollbutton.offsetWidth>0);
+          });
+          
+          test('when scroll-offset is 0, then the left scrollbutton is faded out', function(done) {
+         	  
+            Polymer.Base.async(function() {
+              assert.equal(tabsContainer.scrollLeft, 0);
+              assert.isTrue(leftScrollbutton.classList.contains('not-visible'));
+              assert.isFalse(rightScrollbutton.classList.contains('not-visible'));
+              done();
+            }, 100);
+          	
+          });
+          
+          test('when scroll-offset is \'max\', then the right scrollbutton is faded out', function(done) {
+            //Set scrollLeft to a very high number. Will scroll to outermost position.
+            tabsContainer.scrollLeft = 99999;
+       	  
+            Polymer.Base.async(function() {
+              assert.isFalse(leftScrollbutton.classList.contains('not-visible'));
+              assert.isTrue(rightScrollbutton.classList.contains('not-visible'));
+              done();
+            }, 100);
+            	
+          });
+          
+        });
+  
+        suite('zoomed, scrollable paper-tabs', function() {

```

Comment:
This needs to be wrapped in something like
```javascript
if (Object.keys(document.createElement('div').style).indexOf('zoom') >= 0) {
```
so that it doesn't attempt these if the browser doesn't support `zoom`.